src/vsr/free_set.zig

```diff
@@ -76,8 +79,8 @@ pub const FreeSet = struct {
     blocks_released_prior_checkpoint_durability: BlocksReleasedPriorCheckpointDurability,
 
     /// The number of blocks that are reserved, counting both acquired and free blocks
-    /// from the start of `blocks`.
-    /// Alternatively, the index of the first non-reserved block in `blocks`.
+    /// from the start of `blocks_acquired`.
+    /// Alternatively, the index of the first non-reserved block in `blocks_acquired`.
     reservation_blocks: usize = 0,
```

Comment:
In a follow up, let's change these to u64s, as we want to avoid depending on pointer size.